### PR TITLE
Project name  : Porfolio Desktop version 

### PR DESCRIPTION
**Pull request by Ridwanullahi-code**
* Add desktop version 
*  Used media queries to create the responsiveness
*  Change the page layout to the desktop version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -126,29 +133,32 @@ <h1 class="project-title">Multi-Post<br>Stories</h1>
...
+                        <li><a href="https://twitter.com/Ridwanullahi22"><img src="images/twitter.png" alt=""></a></li>
+                        <li><a href="https://github.com/Ridwanullahi-code"><img src="images/medium.png" alt=""></a></li>
+                        
+                    </ul>      
+                </div>
+                <div>
+                    <!-- button to download resume -->
+                    <a class="button btn" href="">Get my resume</a>
+                </div>
             </div>
```
Comment (ichala) [inline index.html:160]: - [ ] This section should be a static HTML with css styling and not an image kindly remove the image and add a list using `<ul>`

image

File: index.html [hunk 2]
```diff
@@ -126,29 +133,32 @@ <h1 class="project-title">Multi-Post<br>Stories</h1>
...
+            
             <!-- area-->
             <div id="skills">
```
Comment (ichala) [inline index.html:163]: - [ ] kindly update your languages section flex direction for frameworks should be row, and try to edit the overall style of the section to match Figma 

Figma Design            |  Your Design
:-------------------------:|:-------------------------:
image  |  image

File: index.html [hunk 3]
```diff
@@ -18,35 +18,42 @@
                 <nav class="navigation_container">
```
Comment (ichala) [inline index.html:18]: - [ ] kindly add the correct padding left and right in the navbar to match figma

image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @Ridwanullahi-code , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Used Media Query✅ 
- Good Pr Description ✅ 

 ## Required Changes ♻️

- [ ] _**check the comments below**_


## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShahierNashaat — APPROVED** [review]
Hi @Ridwanullahi-code, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**